src/profile-logic/process-profile.js

```diff
@@ -1460,11 +1460,12 @@ function _convertGeckoMarkerSchema(
     isStackBased,
   } = markerSchema;
 
-  const fields: GeckoDynamicFieldSchemaData[] = [];
+  const fields: MarkerSchemaField[] = [];
   const staticFields: GeckoStaticFieldSchemaData[] = [];
   for (const f of data) {
     if (f.value === undefined) {
-      fields.push(f);
+      const { key, label, format, hidden } = f;
+      fields.push({ key, label, format, hidden });
```

Comment:
Hmm this creates addional `hidden: undefined` properties in this object, but I guess that's fine.

Reply:
Yeah, that's a good point.